mssql/spicepod.yaml

```diff
@@ -0,0 +1,116 @@
+version: v1beta1
+kind: Spicepod
+name: spicepod-mssql
+
+datasets:
+  - from: mssql:SalesLT.Address
+    name: address
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.BuildVersion
+    name: build_version
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.Customer
+    name: customer
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.CustomerAddress
+    name: customer_address
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.database_firewall_rules
+    name: database_firewall_rules
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.ErrorLog
+    name: error_log
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.Product
+    name: product
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.ProductCategory
+    name: product_category
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.ProductDescription
+    name: product_description
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.ProductModel
+    name: product_model
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.ProductModelProductDescription
+    name: product_model_product_description
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.SalesOrderDetail
+    name: sales_order_detail
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.SalesOrderHeader
+    name: sales_order_header
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.vGetAllCategories
+    name: v_get_all_categories
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.vProductAndDescription
+    name: v_product_and_description
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+  - from: mssql:SalesLT.vProductModelCatalogDescription
+    name: v_product_model_catalog_description
+    params: 
+      mssql_connection_string: ${ secrets:SPICE_MSSQL_CONNECTION_STRING }
+
+embeddings:
+  - name: openai_embed
+    from: openai
+    params:
+      openai_api_key: ${ secrets:SPICE_OPENAI_API_KEY }
+
+  - name: hf_minilm
+    from: huggingface:huggingface.co/sentence-transformers/all-MiniLM-L6-v2
+
+models:
```

Comment:
We should have two versions of this. One with models and one without.

Reply:
Agreed, will create a ticket for me to split off a quickstart with the models included